mne/io/ctf/eeg.py

```diff
@@ -49,3 +53,46 @@ def _read_eeg(directory):
                     eeg['np'] += 1
     logger.info('    Separate EEG position data file read.')
     return eeg
+
+
+def _read_pos(directory, transformations):
+    """Read the .pos file and return eeg positions as digitizer extra points.
+    """
+    fname = [join(directory, f) for f in listdir(directory) if
+             f.endswith('.pos')]
+    if len(fname) < 1:
+        return list()
+    elif len(fname) > 1:
+        warnings.warn('    Found multiple pos files. Extra digitizer points '
+                      'not added.')
+        return list()
+    logger.info('    Reading digitizer points from %s...' % fname)
+    if transformations['t_ctf_head_head'] is None:
+        warnings.warn('    No transformation found. Extra digitizer ponts not '
+                      'added')
+        return list()
+    fname = fname[0]
+    digs = list()
+    i = 1000
```

Comment:
And the data looks like this:
```
1       Cz      5.29886357      -0.39211620     13.97175211
2       Pz      -2.72999896     0.56457819      14.80592908
3               9.47286768      0.09984190      -2.22056385
```
So the ids would become 1001, 1002, 1003 and so on.